ironfish/src/workerPool/tasks/workerMessages.test.perf.ts

```diff
@@ -48,7 +48,11 @@ describe('WorkerMessages', () => {
     })
 
     expect(true).toBe(true)
-    printResults('createMinersFeeRequest', runs, segment)
+    const { min, max, avg } = printResults('createMinersFeeRequest', runs, segment)
+
+    expect(max).toBeLessThanOrEqual(1.191)
```

Comment:
These numbers are the baseline numbers, basically the number I got when I run these tests in CI jobs and locally.
If the performance is greatly improved by a certain PR, we would want to change the numbers accordingly. I imagine this will not super frequent, maybe once in a quarter.

For the flaky failures, I want to make the perf test as optional so it does not block PR. But we kept the records and can easily identify PRs that  increase the latency.